tests/unit/amo/components/TestUserProfile.js

```diff
@@ -178,6 +178,61 @@ describe(__filename, () => {
     expect(root.find('.UserProfile-homepage')).toHaveLength(0);
   });
 
+  it("renders the user's occupation", () => {
+    const occupation = 'some occupation';
+
+    const { store } = dispatchSignInActions({
+      userProps: {
+        occupation,
+        username: 'tofumatt',
+      },
+    });
+    const root = renderUserProfile({ store });
+
+    expect(root.find('.UserProfile-occupation')).toHaveLength(1);
+    expect(root.find('.UserProfile-occupation').children())
+      .toHaveText(occupation);
+  });
+
+  it("omits occupation if the user doesn't have one set", () => {
+    const { store } = dispatchSignInActions({
+      userProps: {
+        occupation: null,
+        username: 'tofumatt',
```

Comment:
do we need username here if it's set up in renderUserProfile?

Reply:
it seems we do need the username here because `renderUserProfile()` we want a user named `tofumatt`, because that's the default user loaded by `renderUserProfile()`.